### PR TITLE
Make code compatible with realtime submission

### LINKED ISSUE #17: make compatible code with realtime submission

### PR DESCRIPTION
**Pull request by pongsaphol**

### CODE REVIEW

**Review by iammarkps — COMMENTED** [review]
Can you elaborate what tmpID for? thx!

**Comment by pongsaphol** [comment]
There is a problem with some people want to set username to the same name and I need to change their name to random from the crypto library.

**Comment by iammarkps** [comment]
So, this one is for random name for submission list, etc. Not realtime right?

**Comment by pongsaphol** [comment]
Yeah

**Comment by iammarkps** [comment]
This code is currently check only for missing username fetch from submission list right?

File: functions/src/submissions.ts [hunk 1]
```diff
@@ -140,7 +141,6 @@ export const getSubmission = functions
         }
 
         return {
-          ...submission,
```
Comment (pongsaphol) [inline functions/src/submissions.ts:143]: I have removed submission for compatibility with front-end code.

**Comment by iammarkps** [comment]
If I fetch from submission page will it change username? (not submission list)

**Comment by pongsaphol** [comment]
Yes, it will be changed only it has a problem.

**Comment by pongsaphol** [comment]
In this case, there is a problem only in getSubmissionList functions.

**Review by iammarkps — APPROVED** [review]

**Pull request merged**